### PR TITLE
Adding Cisco s350 compliance

### PR DESCRIPTION
**Pull request by nniehoff**
This PR Adds compliance parsing for the cisco s350 model switches.  For the purposes of this module they are identical to the cisco ios devices.

### CODE REVIEW

File: netutils/config/compliance.py [hunk 1]
```diff
@@ -9,6 +9,7 @@
     "cisco_ios": parser.IOSConfigParser,
     "cisco_nxos": parser.NXOSConfigParser,
     "cisco_aireos": parser.AIREOSConfigParser,
+    "ciscos350": parser.IOSConfigParser,
```
Comment (itdependsnetworks) [inline netutils/config/compliance.py:12]: Can we subclass `parser.S300ConfigParser` from  `parser.IOSConfigParser` and change `ciscos350` to `cisco_s300` to match netmiko's defintion of this.

Comment (nniehoff) [reply netutils/config/compliance.py:12]: Sounds good, the napalm module is s350... ugh... updating now

**Comment by jeffkala** [comment]
@nniehoff can you add tests

**Comment by jeffkala** [comment]
bump @nniehoff

**Comment by itdependsnetworks** [comment]
Hey, going to close for now, feel free to open up once updated

**Pull request closed without merging**